File: backend/src/main/java/com/company/model/Product.java
package com.company.model;

public class Product {
    private Integer objectId;
    private String objectName;
    private String typeName;
    private Integer price;
    private Integer orderId;

    private Integer quantity;

    public Integer getOrderId() {
        return orderId;
    }

    public void setOrderId(Integer orderId) {
        this.orderId = orderId;
    }

    public Product(Integer objectId, Integer price) {
        this.objectId = objectId;
        this.price = price;
    }

    public Product(Integer objectId, String objectName, String typeName, Integer orderId) {
        this.objectId = objectId;
        this.objectName = objectName;
        this.typeName = typeName;
        this.orderId = orderId;
    }

    public Product(Integer objectId, String objectName, String typeName) {
        this.objectId = objectId;
        this.objectName = objectName;
        this.typeName = typeName;
    }

    public Integer getPrice() {
        return price;
    }

    public void setPrice(Integer price) {
        this.price = price;
    }

    public Integer getObjectId() {
        return objectId;
    }

    public void setObjectId(Integer objectId) {
        this.objectId = objectId;
    }
    public Product () {}

    public String getObjectName() {
        return objectName;
    }

    public void setObjectName(String objectName) {
        this.objectName = objectName;
    }

    public String getTypeName() {
        return typeName;
    }

    public void setTypeName(String typeName) {
        this.typeName = typeName;
    }

    public Integer getQuantity() {
        return quantity;
    }

    public void setQuantity(Integer quantity) {
        this.quantity = quantity;
    }
}

File: frontend/internet-shop-front/src/app/model/product.ts
export class Product {
  get quantity(): number {
    return this._quantity;
  }

  set quantity(value: number) {
    this._quantity = value;
  }
  get orderId(): number {
    return this._orderId;
  }

  set orderId(value: number) {
    this._orderId = value;
  }
  get objectId(): number {
    return this._objectId;
  }

  set objectId(value: number) {
    this._objectId = value;
  }
  get objectName(): string {
    return this._objectName;
  }

  set objectName(value: string) {
    this._objectName = value;
  }
  get typeName(): string {
    return this._typeName;
  }

  set typeName(value: string) {
    this._typeName = value;
  }
  get price(): number {
    return this._price;
  }

  set price(value: number) {
    this._price = value;
  }

  // @ts-ignore
  private _orderId : number;
  // @ts-ignore
  private _objectId : number;
  // @ts-ignore
  private _objectName : string;
  // @ts-ignore
  private _typeName : string;
  // @ts-ignore
  private _price : number;
  // @ts-ignore
  private _quantity : number;

}
